src/layout/layout.js

```diff
@@ -190,27 +190,33 @@
    *
    * @private
    */
-  MaterialLayout.prototype.screenSizeHandler_ = function() {
-    if (this.screenSizeMediaQuery_.matches) {
-      this.element_.classList.add(this.CssClasses_.IS_SMALL_SCREEN);
 
-      if (this.drawer_) {
-        this.drawer_.setAttribute('aria-hidden', 'true');
-      }
-    } else {
-      this.element_.classList.remove(this.CssClasses_.IS_SMALL_SCREEN);
-      // Collapse drawer (if any) when moving to a large screen size.
-      if (this.drawer_) {
-        this.drawer_.classList.remove(this.CssClasses_.IS_DRAWER_OPEN);
-        this.obfuscator_.classList.remove(this.CssClasses_.IS_DRAWER_OPEN);
-
-        if (this.element_.classList.contains(this.CssClasses_.FIXED_DRAWER)) {
-          this.drawer_.setAttribute('aria-hidden', 'false');
+  function screenSizeHandler(e) {
+    // modified to query dependent elements rather than binding materialLayout to windows media query result
+    var materialLayout = window.document.querySelector('.mdl-layout')
```

Comment:
I forgot to mention before, but should we handle multiple materialLayout elements in the dom? Seems kind of strange, but might happen.
I guess we could use querySelectorAll here and loop. In that case we could probably set a single global media query instead of making one for each MaterialLayout.

Reply:
Yeah for sure. I wasn't sure how far to go with patch the problem, and/or support the future of 'derivita:mdl'

Will do!